docs/site/Relation-generator.md

```diff
@@ -86,7 +109,12 @@ HasMany, HasOne, or BelongsTo relations.
 lb4 relation --sourceModel=<sourceModel>
 --destinationModel=<destinationModel> --throughModel=<throughModel>
 --relationType=<hasManyThrough> [--relationName=<relationName>]
+[--sourceModelPrimaryKey=<sourceModelPrimaryKey>]
```

Comment:
For `HasManyThrough`, is `sourceModelPrimaryKey` the same as `sourceKeyOnThrough` and `destinationModelPrimaryKey` the same as `targetKeyOnThrough`?

Reply:
Good question! Indeed they refer to the same thing, but they may be different. For example, `sourceModelPrimaryKey` on `Customer` is `id`, and `sourceKeyOnThrough` on `CustomerOrderLink` defaults to `customerId`, and `targetKeyOnThrough` defaults to `orderId`. Therefore those two params are needed IMO.